txt2tags.py

```diff
@@ -1189,6 +1195,60 @@ def getTags(config):
             # TODO: placeholder (mark for unknown syntax)
             # if possible: http://www.wikicreole.org/wiki/Placeholder
         },
+            # regular markdown: http://daringfireball.net/projects/markdown/syntax
+            # markdown extra:   http://michelf.com/projects/php-markdown/extra/
+        "md": {
+            "title1"               : "# \a "         ,
+            "title2"               : "## \a "        ,
+            "title3"               : "### \a "       ,
+            "title4"               : "#### \a "      ,
+            "title5"               : "##### \a "     ,
+            "blockVerbLine"        : "    "          ,
+            "blockQuoteLine"       : "> "            ,
+            "fontMonoOpen"         : "`"             ,
+            "fontMonoClose"        : "`"             ,
+            "fontBoldOpen"         : "**"            ,
+            "fontBoldClose"        : "**"            ,
+            "fontItalicOpen"       : "*"             ,
+            "fontItalicClose"      : "*"             ,
+            "fontUnderlineOpen"    : ""              ,
+            "fontUnderlineClose"   : ""              ,
```

Comment:
Instead of dropping the underscores, I propose to simply preserve the double underscores.

Reply:
I don't know how to handle this. Markdown doesn't support underscores, as far as I know, maybe some derivatives do.
I propose to merge the pull request, and after that you do as you wish.